package.json

```diff
@@ -1,17 +1,40 @@
 {
-    "name": "",
-    "version": "",
-    "description": "",
-    "author": "",
-    "license": "",
-    "engines": {
-      "node": ">=6.13.0"
-    },
-    "scripts": {
-    },
-    "dependencies": {
-    },
-    "devDependencies": {
-      "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
-    }
-  }
\ No newline at end of file
+  "name": "reviews",
+  "version": "",
+  "description": "Front end capstone project",
```

Comment:
@janicelam21 

This should be a description of what the reviews component will do when completed!

Reply:
modified this!